server/interceptor/interceptor.go

```diff
@@ -141,6 +149,10 @@ func (tp *trillianProcessor) Before(ctx context.Context, req interface{}) (conte
 		incRequestDeniedCounter(badInfoReason, 0, quotaUser)
 		return ctx, err
 	}
+	if err = ctx.Err(); err != nil {
```

Comment:
FYI, this is all in-memory processing. If the ctx is expired here, chances are it was already expired (or the deadline is way too low)

Reply:
The log message about "read tree info" was confusing. I'll probably remove this check then.

Reply:
Removed.